compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1432,12 +1432,12 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsDivCheck() {return _flags1.testAny(SupportsDivCheck);}
    void setSupportsDivCheck() {_flags1.set(SupportsDivCheck);}
 
-   bool getSupportsArrayCopy() {return _flags1.testAny(SupportsArrayCopy);}
-   void setSupportsArrayCopy() {_flags1.set(SupportsArrayCopy);}
-
    bool getSupportsPrimitiveArrayCopy() {return _flags2.testAny(SupportsPrimitiveArrayCopy);}
    void setSupportsPrimitiveArrayCopy() {_flags2.set(SupportsPrimitiveArrayCopy);}
 
+   bool getSupportsReferenceArrayCopy() {return _flags2.testAny(SupportsReferenceArrayCopy);}
```

Comment:
Argh, this is such a silly but very costly mistake. I'm very glad we do thorough code reviews and I appreciate your level of detail. This is a bug. Fixed in 13f34f045b33633a1a598a18000da9a21cf328dc.